action/mail.php

```diff
@@ -0,0 +1,205 @@
+<?php
+/**
+ * @license GNU General Public License, version 2
+ */
+
+
+if (!defined('DOKU_INC')) die();
+
+
+/**
+ * Class action_plugin_publish_mail
+ *
+ * @author Michael Große <grosse@cosmocode.de>
+ */
+class action_plugin_publish_mail extends DokuWiki_Action_Plugin {
+
+    /**
+     * @var helper_plugin_publish
+     */
+    private $hlp;
+
+    function __construct() {
+        $this->hlp = plugin_load('helper','publish');
+    }
+
+    public function register(Doku_Event_Handler $controller) {
+
+        $controller->register_hook('IO_WIKIPAGE_WRITE', 'AFTER', $this, 'send_change_mail', array());
+    }
+
+    /**
+     * send an email to inform about a changed page
+     *
+     * @param $event
+     * @param $param
+     * @return bool|mixed
```

Comment:
some short description about the parameters would be helpful. especially for the return value here